apps/els_lsp/priv/code_navigation/src/code_navigation.erl

```diff
@@ -122,3 +122,9 @@ macro_b(_X, _Y) ->
 
 function_mb() ->
   ?MACRO_B(m, b).
+
+code_navigation() -> code_navigation.
```

Comment:
It looks like there's a case which doesn't work as expected. Try to tweak this test code by using an atom referring to a different module name. Eg:

```
code_navigation_atoms() -> code_navigation_atoms.

code_navigation_atoms(X) -> X.

multiple_instances() -> {code_navigation_atoms, [simple_list], "abc"}.
```
Only the functions will be found.

Reply:
Many thanks for spotting this. I believe the issue has now been fixed.